### DISCUSSION TITLE
关于手机浏览器使用该项目时退出问题

### DISCUSSION THREAD

**Discussion by Funyu78** [GPTGenius/chatgpt-vercel#75] [Ideas]
在手机上使用右滑直接退出而不是返回上一层真的难受，另外在设置里建议添加一个确认提示保存成功的按钮每次点×没反馈也很难受 😎😎😎

**Comment by lvqq** [comment]
点×没反馈 是指什么？

**Reply by Funyu78** [reply]
没有提示一下“完成”感觉有点难受，我强迫症哈哈哈，感觉在下面添加一个确认的按钮好看一点

**Comment by lvqq** [comment]
https://github.com/GPTGenius/chatgpt-vercel/pull/79 中已支持
